ert_storage/endpoints/records.py

```diff
@@ -391,3 +391,9 @@ def get_ensemble_records(
         rec.name: js.RecordOut(id=rec.id, name=rec.name, data=rec.data)
         for rec in ensemble.records
     }
+
+
+@router.get("/records/{record_id}", response_model=js.RecordOut)
```

Comment:
What are these `records`? Are they bound to a single experiment or do we assume all records having unique ids among experiments?

Reply:
all records have a unique id in the records table, currently stored as an incremental integer, but the case could be made for using something like uuid4 for all our id-columns in our database